modules/link-in-bio/module.php

```diff
@@ -41,4 +41,27 @@ public static function get_experimental_data(): array {
 			],
 		];
 	}
+
+	public function __construct() {
+		parent::__construct();
+
+		add_action( 'wp_enqueue_scripts', [ $this, 'register_style' ] );
+	}
+
+	/**
+	 * Register styles.
+	 *
+	 * At build time, Elementor compiles `/modules/link-in-bio/assets/scss/frontend.scss`
+	 * to `/assets/css/widget-link-in-bio.min.css`.
+	 *
+	 * @return void
+	 */
+	public function register_style() {
```

Comment:
Maybe we need a better solution for grouped widgets? 🤔

Reply:
@louiswol94 What do you suggest?

Reply:
@rami-elementor Let's leave it for now :)